chartfx-math/src/test/java/de/gsi/math/FastMathTests.java

```diff
@@ -22,87 +22,87 @@
+            final float val2 = MathBaseFast.cosDeg((float) i);
+            final float diffApproximation = java.lang.Math.abs(val1 - val2);
             assertTrue(diffApproximation <= 2e-3, i + ": cos(" + x + ") diff (float, deg) = " + diffApproximation);
         }
 
         for (int i = -360; i < 360; i++) {
-            final double x = Math.toRadians(i);
-            final double val1 = Math.cos(x);
-            final double val2 = FastMath.cosDeg((double) i);
-            final double diffApproximation = Math.abs(val1 - val2);
+            final double x = java.lang.Math.toRadians(i);
+            final double val1 = java.lang.Math.cos(x);
+            final double val2 = MathBaseFast.cosDeg((double) i);
+            final double diffApproximation = java.lang.Math.abs(val1 - val2);
             assertTrue(diffApproximation <= 2e-3, i + ": cos(" + x + ") diff (double, deg) = " + diffApproximation);
         }
     }
 
     @Test
     public void simpleTest() {
-        final int oldPrecision = FastMath.getPrecision();
-        FastMath.setPrecision(100);
-        assertEquals(100, FastMath.getPrecision(), "set precision");
+        final int oldPrecision = MathBaseFast.getPrecision();
+        MathBaseFast.setPrecision(100);
+        assertEquals(100, MathBaseFast.getPrecision(), "set precision");
 
-        FastMath.setPrecision(oldPrecision);
+        MathBaseFast.setPrecision(oldPrecision);
 
         assertThrows(IllegalArgumentException.class, () -> {
-            FastMath.setPrecision(-1);
+            MathBaseFast.setPrecision(-1);
         });
     }
 
     @Test
     public void sineIdentityChecks() {
-        double precision = FastMath.getPrecision();
-        int range = FastMath.getPrecision();
+        double precision = MathBaseFast.getPrecision();
+        int range = MathBaseFast.getPrecision();
         for (int i = -range; i < range; i++) {
-            final float x = (float) (2.0 * Math.PI * i / precision);
-            final float val1 = (float) Math.sin(x);
-            final float val2 = FastMath.sin(x);
-            final float diffApproximation = Math.abs(val1 - val2);
+            final float x = (float) (2.0 * java.lang.Math.PI * i / precision);
+            final float val1 = (float) java.lang.Math.sin(x);
+            final float val2 = MathBaseFast.sin(x);
+            final float diffApproximation = java.lang.Math.abs(val1 - val2);
             assertTrue(diffApproximation <= 1e-3, i + ": cos(" + x + ") diff (float) = " + diffApproximation);
         }
 
         for (int i = -range; i < range; i++) {
-            final double x = 2.0 * Math.PI * i / precision;
-            final double val1 = Math.sin(x);
-            final double val2 = FastMath.sin(x);
-            final double diffApproximation = Math.abs(val1 - val2);
+            final double x = 2.0 * java.lang.Math.PI * i / precision;
+            final double val1 = java.lang.Math.sin(x);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.sin' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)